.github/workflows/publish-demo.yml

```diff
@@ -36,30 +36,20 @@ jobs:
           git-lfs: 'true'
       - name: 🚚 Pull Git LFS files
         run: git lfs pull
-
-      #  Setup Expo CLI action
-      # - name: Setup Expo
-      #   uses: expo/expo-github-action@v5
-      #   with:
-      #     expo-version: 3.x
-      #     expo-username: ${{ secrets.EXPO_CLI_USERNAME }}
-      #     expo-password: ${{ secrets.EXPO_CLI_PASSWORD }}
-      #     expo-cache: true
-
+      - name: Setup Expo
+        uses: expo/expo-github-action@v7
+        with:
+          expo-version: latest
+          eas-version: latest
+          token: ${{ secrets.COMMUNITY_EXPO_ACCESS_TOKEN }}
       - name: 🧶 Install workspace node modules
         if: steps.expo-caches.outputs.yarn-workspace-hit != 'true'
         run: yarn install --frozen-lockfile
       - name: 🔧 Install Expo CLI v3
         run: yarn global add expo-cli@3
```

Comment:
AFAIK the Expo GitHub Action installs Expo CLI (a newer version too). So I think we want to remove this step.

Also we might want to use the new CLI soon (npx expo) though I don't think it needs to be part of this PR. https://blog.expo.dev/new-versioned-expo-cli-cf6e10632656